smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/DocumentMemberDeserVisitor.java

```diff
@@ -95,10 +128,29 @@ public DocumentMemberDeserVisitor(
      *
      * @return The data source.
      */
-    protected String getDataSource() {
+    protected final String getDataSource() {
         return dataSource;
     }
 
+    /**
+     * Gets the default timestamp format used in absence of a TimestampFormat trait.
+     *
+     * @return The default timestamp format.
+     */
+    protected final Format getDefaultTimestampFormat() {
+        return defaultTimestampFormat;
+    }
+
+    /**
```

Comment:
We probably need to make a guarantee in the API documentation that the Set is ordered. Otherwise, the generated code would be random.